fastlane/swift/formatting/Rakefile

```diff
@@ -8,6 +8,7 @@ task(:brew) do
   raise '`brew` is required. Please install brew. https://brew.sh/' unless system('which brew')
 
   puts('➡️  Brew')
+  sh('brew update')
```

Comment:
This is CI fix change.

Reply:
I thought about this for a second and thought that maybe we should be checking the version elsewhere and in some other way that is more stable and reproducible than this, however, since this is just an internal script used for formatting fastlane.swift, I think it's fine to leave the generic "update" command here 🙏 

Thanks for fixing this @TLizer 💪